.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,23 +1,45 @@
-## Description
-- Brief description
-- About what's being done in this PR
-- And why are those changes being done
+## Summary
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)